planning/behavior_path_planner/src/utils/utils.cpp

```diff
@@ -1935,7 +1935,7 @@ void makeBoundLongitudinallyMonotonic(
 
       const auto theta = normalizeRadian(p1_to_p2 - p2_to_p3);
 
-      return (is_points_left && 0 < theta) || (!is_points_left && theta < 0);
+      return (is_points_left && 0 > theta) || (!is_points_left && theta > 0);
```

Comment:
@mehmetdogru Btw, I might have found a bug. If the **LEFT** bound is monotonic, `theta` will be smaller than zero. On the other hand, if the **RIGHT** bound is monotonic, `theta` will be larger than zero.

Therefore, I guess we should fix it as follow:

```suggestion
      return (is_points_left && 0 < theta) || (!is_points_left && theta > 0);
```

Reply:
Thanks @satoshi-ota -san. I think it is already how you want it to be. What you suggest with the code change is:

If the **LEFT** bound is monotonic, theta will be **larger** than zero. On the other hand, if the **RIGHT** bound is monotonic, theta will be **larger** than zero

Reply:
> If the LEFT bound is monotonic, theta will be larger than zero. On the other hand, if the RIGHT bound is monotonic, theta will be larger than zero

Ah! Sorry, it was a misjudgement :pray: